src/hotspot/share/gc/shenandoah/shenandoahScanRemembered.inline.hpp

```diff
@@ -0,0 +1,401 @@
+/*
+ * Copyright (c) Amazon.com, Inc. or its affiliates.  All rights reserved.
```

Comment:
Missing copyright year.

Reply:
We've asked around inside Amazon.  We're good to add in the copyright dates.  So next commit will include these.